common/prometheus-server-pre7/values.yaml

```diff
@@ -353,11 +353,12 @@ serviceDiscoveries:
     enabled: false
     # Full list of APIserver metrics: https://github.com/kubernetes/apiserver/blob/master/pkg/endpoints/metrics/metrics.go .
     allowedMetrics:
-      - apiserver_request_count
-      - apiserver_request_latencies_bucket
       - apiserver_request_total
-      - apiserver_request_latency_seconds
+      - apiserver_longrunning_requests
+      - apiserver_request_duration_seconds
       - apiserver_dropped_requests_total
+      - apiserver_requested_deprecated_apis
+      - apiserver_current_inflight_requests
       - etcd_request_latency_seconds
       - etcd_object_counts
```

Comment:
Since 1.21
> Rename metrics etcd_object_counts to apiserver_storage_object_counts and mark it as stable. The original etcd_object_counts metrics name is marked as "Deprecated" and will be removed in the future.